keras/utils/training_utils.py

```diff
@@ -101,6 +101,14 @@ def multi_gpu_model(model, gpus):
     if K.backend() != 'tensorflow':
         raise ValueError('`multi_gpu_model` is only available '
                          'with the TensorFlow backend.')
+
+    if not gpus:
```

Comment:
Below there is line `available_devices = _get_available_devices()`.

Move this line to line 104 and use `available_devices` in your code, this will avoid calling this function twice